packages/react-baseline-inputs/src/Field.tsx

```diff
@@ -76,6 +77,7 @@ export function Field<T extends FieldProps>({
   const inputProps = {
     id,
     className: inputClassName,
+    "aria-labelledby": error ? `${id} ${errorLabelId}` : undefined,
```

Comment:
Should this just be:

```suggestion
    "aria-labelledby": error ? errorLabelId : undefined,
```

Reply:
Oh, okay! I didn't realize the intent was to specify multiple labels. That's cool!